qa-admin/src/test/java/ru/volpi/qaadmin/web/controller/CategoriesRestControllerTest.java

```diff
@@ -113,6 +119,24 @@ void triesCreateEmptyCategory() throws Exception {
         assertThat(content).contains(CATEGORY_CANNOT_BE_EMPTY);
     }
 
+    @Test
+    @WithMockUser("admin")
+    @DisplayName("Creates category with bad name")
+    void triesCreateBigCategory() throws Exception {
+        this.mockMvc.perform(
+                put(CATEGORIES)
+                    .contentType(MediaType.APPLICATION_JSON)
+                    .content(
+                        """
+                            {
+                                "name": "%s"
+                            }
+                            """.formatted(BAD_CATEGORY_NAME)
+                    )
+                    .characterEncoding(StandardCharsets.UTF_8)
+            ).andExpect(status().isCreated());
+    }
+
```

Comment:
@Andrew-Sem according to this, there is nothing wrong in server logic with that name, wdyt

Reply:
@l3r8yJ it works good for me now, even in master. I have no idea how did I catch this before